site/topics/numbers/number-bases.rst

```diff
@@ -2,11 +2,251 @@
 Number Bases
 ************
 
+* Have you ever *really* thought about counting?
+
 
 
 Decimal (Base Ten)
 ==================
 
+* The decimal number system is used in everyday life
+* The decimal number system is based ten; it consists of ten different symbols/numerals
+
+    * :math:`0, 1, 2, 3, 4, 5, 6, 7, 8, 9`
+
+
+* These symbols have an ordering to them
+
+    * :math:`0 \rightarrow 1 \rightarrow  2 \rightarrow  3 \rightarrow  4 \rightarrow  5 \rightarrow  6 \rightarrow  7 \rightarrow  8 \rightarrow  9`
+
+
+* These symbols also have a meaning/value associated with them
+
+    * The symbol :math:`5` means *five*
+    * It conveys a magnitude
+
+
+* The magnitude of these values correspond to the ordering
+
+    * :math:`0 < 1 <  2 <  3 <  4 <  5 <  6 <  7 <  8 <  9`
+
+
+
+Counting in Base Ten
+--------------------
+
+* Counting in base ten consists of moving to the next symbol
+
+    .. list-table:: Counting With Nine Symbols
+        :widths: 50 50
+
+        * - :math:`0`
+          - Zero
+        * - :math:`1`
+          - One
+        * - :math:`2`
+          - Two
+        * - :math:`3`
+          - Three
+        * - :math:`4`
+          - Four
+        * - :math:`5`
+          - Five
+        * - :math:`6`
+          - Six
+        * - :math:`7`
+          - Seven
+        * - :math:`8`
+          - Eight
+        * - :math:`9`
+          - Nine
+        * - :math:`????`
+          - Ten?
+
+
+* However, there are only ten unique symbols
+* This means it's not possible to count past nine with the symbols alone
+* This is where symbol position comes in
+
+
+Position
+^^^^^^^^
+
+* First, consider that each number has an infinite number of zeros in front of it
+
+    * Why this is will be discussed shortly
+
+
+* Consider the below table where only seven of the leading zeros are included
+
+    * Thus, these numbers will each have a total of eight symbols
+    * Each of these symbols/numerals, in this context, is called a *digit*
+
+        * Digit comes from latin (*digitus*), meaning finger or toe, of which, humans typically have ten of each
+
+
+    .. list-table:: Values expressed with eight digits
+        :widths: 50 50
+
+        * - :math:`00000000`
+          - Zero
+        * - :math:`00000001`
+          - One
+        * - :math:`00000002`
+          - Two
+        * - :math:`00000003`
+          - Three
+        * - :math:`00000004`
+          - Four
+        * - :math:`00000005`
+          - Five
+        * - :math:`00000006`
+          - Six
+        * - :math:`00000007`
+          - Seven
+        * - :math:`00000008`
+          - Eight
+        * - :math:`00000009`
+          - Nine
+
+
+* To count past the value nine, a new rule is introduced
+* If the no more symbols are left
+
+    * Reset the symbol to zero in the current position
+    * Change to the subsequent symbol in the next position
+
+
+    .. list-table:: Counting with eight digits
+        :widths: 50 50
+
+        * - :math:`00000000`
+          - Zero
```

Comment:
Can probably just start counting as `...` then 9, no need to duplicate so much. The prior table is still on the screen.